versioned_sidebars/version-4.x-sidebars.json

```diff
@@ -1,12 +1,12 @@
 {
-  "version-4.x/docs": [
```

Comment:
Is this something that changed with docusaurus@3?

Reply:
Yes, ths is not needed or supported anymore: https://github.com/facebook/docusaurus/pull/9310.